tasks/dispatcher_configs.yml

```diff
@@ -33,7 +18,6 @@
     - '{{ dispatcher_config_set }}'
   become: True
   register: result
-  when: ansible_os_family == "RedHat"
 
 - name: restart apache
```

Comment:
Why do we need this strange task doing nothing but force triggering a handler?

Reply:
replaced to other task